src/styles/blocks/card.scss

```diff
@@ -0,0 +1,143 @@
+
+/* #region main */
+
+.cards {
```

Comment:
cards and card are different BEM-blocks, so they need to be put in different files

Reply:
done